api/tests/unit/domain/services/solution-service-qroc_test.js

```diff
@@ -8,18 +8,40 @@ const ANSWER_OK = AnswerStatus.OK;
 
 describe('Unit | Service | SolutionServiceQROC ', function() {
 
+  describe('match, with numerical answers and solutions', function() {
+    [
+      { case:'when two numbers are strictly equal', answer: '0.123', solution: '0.123', expectedAnswerStatus: ANSWER_OK },
+      { case:'when 0 is added in front of the answer', answer: '0123', solution: '123', expectedAnswerStatus: ANSWER_OK },
```

Comment:
**suggestion:** J'ajouterais un test multiple : 

      { case:'(multiple solutions) answer is not the first propose solution', answer: '23', solution: '21\n22\n23\n', expectedAnswerStatus: ANSWER_OK  },